onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -336,8 +321,8 @@ def get_qmin_qmax_for_qType(qType, reduce_range=False, symmetric=False):  # noqa
 
     if reduce_range:
         qrange = ONNX_INT_TYPE_REDUCED_RANGE.get(qType)
-    elif symmetric:
-        qrange = ONNX_INT_TYPE_SYMMETRIC_RANGE.get(qType)
+    elif symmetric and qType in ONNX_INT_TYPE_SYMMETRIC_RANGE:
```

Comment:
this makes sense. however, it's slightly different from the original behavior - if `qType` is unsigned and `symmetric` is True then it is now an error. not sure whether anyone is relying on the original behavior.

at least maybe we can have a specific error message about the type not supporting `symmetric == True`

Reply:
Hmm I don't see this. If `qType` is unsigned, then the else branch is taken and we get the range from `ONNX_INT_TYPE_RANGE`. This is equivalent to the original behavior where `symmetric` was essentially ignored for unsigned types.

Reply:
sorry, you're right. I got confused looking at the diff.